### ISSUE TITLE
Support: Fetching events from server 

### ISSUE THREAD

**Issue by 0xffca** [MISP/PyMISP#226]
Hello! 
Is there a way to fetch events from a server by referencing the server_id as for feeds in the example:
fetch_events_feed.py
Thanks in advance!

**Comment by Rafiot** [comment]
I don't think it is possible, and I'm also not sure it is doable. @iglocska what do you think?
